helm/hpcc/values.yaml

```diff
@@ -38,63 +38,75 @@ global:
     moneyLocale: "en_US.UTF-8"
     # perCpu: 0.126
 
-## placement
+## placements
 ## The placement is responsible for finding the best Node for a Pod. It can be configured
 ## through placement to include an array of objects to configure the Kubernetes Scheduler.
-## It must have a "pods" list which tells which pod the settings will be applied to.
-## The list item with suffix "-" and <cluster name> means it is Job which tries to distinguish
-## from the same pod name for Development type.
-## Here are the HPCC Systems Pods' name or prefixs:
-##   eclwatch, hthor, eclservices, eclqueries, esdl-sandbox, sql2ecl
-##   mydali, roxie-workunit, myeclccserver, roxie-toposerver
-##   <cluster name>-thor-eclagent, <cluster name>-thor-thoragent
-##   <cluster name>-hthor-, <cluster-name>-compile-, <cluster name>-thormanager-,
-##   <cluster name>-thorworker-, <cluster name>-roxie-agent-
-## The settings can also be HPCC Systems component types: dali, esp, eclagent, eclccserver,
-## roxie, thor, or a cluster name which is from the array of items of each type.
-## Supported placement configuration:
+## It must have a "pods" list which tells it which pod the settings will be applied to.
+## The synatx is:
+##   placements:
+##   - pods [list]
+##     placement:
+##       <supported configurations>
+##
+## The list item in "pods" can be one of the following:
+## 1) HPCC Systems component types in format: "type:<type name>". <type name> includes
+##    dali, esp, eclagent, eclccserver, roxie, thor. For example "type:esp"
+## 2) Target, the name of array item of above type in format "target:<target name>"
+##    For example "target:roxie", "target:thor".
+## 3) Pod, "Deployment" metadata name, which usually should be from the name of the array
+##    item of a type. For example, "eclwatch", "mydali", "thor-thoragent"..
```

Comment:
not a type "mydali" could be "anything" - these are names.

Reply:
"mydali" is not type but target. But since dali only is allowed a single target (instance) maybe this is bad example.
Let say eclccserver:
eclccserver:
\- name eclccserver-1
....
\- name eclccserver-2

"target:eclccserver-2" will apply only for all pods and jobs in target "eclccserver". and "type: eclccserver" will apply any pod/job defined in eclccserver.yaml